### ISSUE TITLE
Share Map Modal Not Closing

### ISSUE THREAD

**Issue by tannerjt** [Esri/storymap-crowdsource#6]
When running the development application, I am able to open and close the modal window for the 'share' this story map link.  However, when I build the application, I am getting the error:

```
Uncaught TypeError: Cannot read property 'height' of undefined
```
I am assuming this is a result of something gone wrong during the grunt build process.  Looks like it is having problems selecting the DOM element.  Any thoughts?  Anybody having the same problem?

**Comment by tannerjt** [comment]
It appears the problem (or something similar) is also occurring in the 'builder' portion as well.  When trying to edit the title or subtitle fields, this error is being thrown.  Is it possible that Uglify is removing a DOM reference ID or something?

**Comment by ssylvia** [comment]
@tannerjt It appears this is an issue with a third-party library we are using to autosize our text inputs. One of the latest patches they made causes the TypeError in our code. Our bower.json allows for all "patch" updates to be automatically downloaded when clean installing bower dependencies. I've locked down the version to the latest that still works.

There's also another update I just added which fixes the photo upload for iOS 10.3.0 and higher. I recommend merging in the latest release and running `bower install`.
